.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile

```diff
@@ -155,7 +167,7 @@ pipeline {
 def runWith(lib, failedTests, clusterName, stackVersion) {
     sh ".ci/setenvconfig e2e/stack-versions $clusterName $stackVersion"
     script {
-        env.SHELL_EXIT_CODE = sh(returnStatus: true, script: 'make -C .ci get-test-license get-elastic-public-key TARGET=ci-e2e ci')
+        env.SHELL_EXIT_CODE = sh(returnStatus: true, script: "make -C .ci STACK_VERSION=$stackVersion get-test-license get-elastic-public-key TARGET=ci-e2e ci")
```

Comment:
`setenvconfig` already sets the `STACK_VERSION` in the `.env` but the `.ci/Makefile` don't include the `.env`.
It's a bit annoying and prone to be forgotten having to propagate the stack version here.